### PR TITLE
add mysql ssl config

### PR DESCRIPTION
**Pull request by AndrewwHummer**
add mysql ssl config

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]

File: admin_manual/configuration_database/linux_database_configuration.rst [hunk 1]
```diff
@@ -161,6 +161,23 @@ In case of UTF8MB4 you will also find::
...
+  'dbdriveroptions' => [
+    1007 => '/../ssl-key.pem',
+    1008 => '/../ssl-cert.pem',
+    1009 => '/../ca-cert.pem',
+    1014 => true,
+  ],
+  
+Adjust the paths to the pem files for your environment.
+
+``1014 => true`` translates to ``MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => true``
```
Comment (MorrisJobke) [inline admin_manual/configuration_database/linux_database_configuration.rst:180]: Actually not. That is 1016.

File: admin_manual/configuration_database/linux_database_configuration.rst [hunk 2]
```diff
@@ -161,6 +161,23 @@ In case of UTF8MB4 you will also find::
...
+
+This just covers the SSL database configuration on the Nextcloud server. First you need to configure your database server accordingly.
+
+::
+
+  'dbdriveroptions' => [
+    1007 => '/../ssl-key.pem',
+    1008 => '/../ssl-cert.pem',
+    1009 => '/../ca-cert.pem',
+    1014 => true,
```
Comment (MorrisJobke) [inline admin_manual/configuration_database/linux_database_configuration.rst:175]: Also those are all are not the correct ones.

Bildschirmfoto 2020-10-28 um 13 26 55

Comment (AndrewwHummer) [reply admin_manual/configuration_database/linux_database_configuration.rst:175]: Oh dear. Then I am baffeld why it works though....

Comment (MorrisJobke) [inline admin_manual/configuration_database/linux_database_configuration.rst:175]: Let's use the constants directly:

```suggestion
    \PDO::MYSQL_ATTR_SSL_KEY => '/../ssl-key.pem',
    \PDO::MYSQL_ATTR_SSL_CERT => '/../ssl-cert.pem',
    \PDO::MYSQL_ATTR_SSL_CA => '/../ca-cert.pem',
    \PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => true,
```

**Review by MorrisJobke — CHANGES_REQUESTED** [review]
See inline comment

File: admin_manual/configuration_database/linux_database_configuration.rst (hunk 2 above)
Comment (AndrewwHummer) [reply admin_manual/configuration_database/linux_database_configuration.rst:175]: There was an issue with using the constants directly, it  breaks the usage of ssl 
https://github.com/nextcloud/server/issues/19738

this is why i switched to the "numbers" in the first place.

Comment (AndrewwHummer) [reply admin_manual/configuration_database/linux_database_configuration.rst:175]: So at least my ssl config, works with the numeric values, with the constants my nc server does not come up...

File: admin_manual/configuration_database/linux_database_configuration.rst [hunk 3]
```diff
@@ -161,6 +161,23 @@ In case of UTF8MB4 you will also find::
...
+  'dbdriveroptions' => [
+    \PDO::MYSQL_ATTR_SSL_KEY => '/../ssl-key.pem',
+    \PDO::MYSQL_ATTR_SSL_CERT => '/../ssl-cert.pem',
+    \PDO::MYSQL_ATTR_SSL_CA => '/../ca-cert.pem',
+    \PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => true,
+  ],
+  
+Adjust the paths to the pem files for your environment.
+
+``1014 => true`` translates to ``MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => true``
```
Comment (MorrisJobke) [inline admin_manual/configuration_database/linux_database_configuration.rst:180]:
```suggestion
```

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**